src/EventManagement.Services/OrderService.cs

```diff
@@ -100,7 +100,7 @@ public class OrderService : IOrderService {
 
 					Price = variant?.Price ?? product.Price,
 					VatPercent = variant?.VatPercent ?? product.VatPercent,
-					Quantity = variant?.MandatoryCount ?? product.MandatoryCount,
+					Quantity = variant?.MaxOrdersCount ?? product.MaxOrdersCount,
```

Comment:
Is this right?

Reply:
Yes, looks like the model was changed in a previous commit:

https://github.com/gldraphael/EventManagement/blob/0677c79ace2894572b3cd0dac7aca18765674d7c/src/EventManagement.Domain/ProductVariant.cs#L26

Reply:
Fixed in 4cb0792 :)